routes/shared.js

```diff
@@ -35,8 +35,6 @@ var addRoutes = function(app, session) {
     app.get("/shared/showdown.js", sharedFile("showdown/src/showdown.js"));
     app.get("/shared/lodash.js", sharedFile("lodash/lodash.js"));
     app.get("/shared/lodash-min.js", sharedFile("lodash/lodash.min.js"));
-    // TODO serve a minified version of this
-    app.get("/shared/jade-runtime.js", sharedFile("jade/runtime.js"));
```

Comment:
https://pugjs.org/api/reference.html seems to indicate that the Pug runtime is still required?

Reply:
is not necessary, the compile function includes all the necessaries to that, and the new CLI don't have an option for an external runtime: https://github.com/pugjs/pug/issues/2547#issuecomment-273760721

Reply:
Wow, that's pretty annoying that that documentation still isn't up-to-date. Filed pugjs/pug-en#26 to fix that.